easybuild/easyconfigs/l/libwpe/libwpe-1.13.3-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
```

Comment:
please do not strip previous author attributions

Reply:
ok, sorry